PyTado/interface/interface.py

```diff
@@ -51,7 +51,7 @@ def __init__(
         )
 
         if self._http.is_x_line:
-            self._api: API.Tado | API.TadoX = API.TadoX(http=self._http, debug=debug)
+            self._api: API.TadoBase = API.TadoX(http=self._http, debug=debug)
```

Comment:
Should we add this typehint to the else clause too?

Reply:
I did get an error in python 3.11. But we are not using 3.11 anymore, so I added the type hint

Reply:
mypy pre-commit will fail, if I add the seconds type hint:

```
PyTado/interface/interface.py:57: error: Attribute "_api" already defined on line 55  [no-redef]
```

Reply:
Interesting - so it inferred it from above :thinking: